prime-router/src/main/kotlin/azure/LookupTableFunctions.kt

```diff
@@ -50,26 +41,30 @@ class LookupTableFunctions(
             route = "lookuptables/list"
         ) request: HttpRequestMessage<String?>
     ): HttpResponseMessage {
-        return getOktaAuthenticator().checkAccess(request) {
-            try {
-                val showInactive = request.queryParameters[showInactiveParamName]
-                    ?.equals("true", true) ?: false
-                // Return everything if showInactive is true
-                val list = lookupTableAccess.fetchTableList(showInactive)
-                when (request.httpMethod) {
-                    HttpMethod.HEAD -> {
-                        // Get the latest date of all the tables
-                        val lastModified = list.maxWithOrNull(compareBy { it.createdAt })?.createdAt
-                        HttpUtilities.okResponse(request, lastModified)
-                    }
-                    else -> {
-                        HttpUtilities.okResponse(request, mapper.writeValueAsString(list))
-                    }
+        // Do authentication
+        val claims = AuthenticatedClaims.authenticate(request)
+            ?: return HttpUtilities.unauthorizedResponse(request, authenticationFailure)
+        // No authorization check!   Anyone authenticated in ReportStream is allowed to access this function.
+        logger.info("User ${claims.userName} is authorized for endpoint ${request.uri}")
```

Comment:
would it be more correct to say `<user> is authenticated` since we aren't running through any authorization? In English it comes across fine but wonder if it is misleading in terms of engineering jargon

Reply:
I think this might be ok - its says, "the fact that the user is athenticated means that they are also authorized."